rakelib/user_email_identifier.rake

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'roo'
+
+desc 'Build a CSV of user email identifiers'
+
+namespace :user_email_identifier, type: :task do
+  task :build_csv, %i[uuid_list] => :environment do |_, args|
```

Comment:
I was running into errors when trying to execute this locally. I had to make this change to get it working:

```ruby
namespace :user_email_identifier do
  desc 'Build a CSV of user email identifiers'
  task :build_csv, %i[uuid_list] => :environment do |_, args|
```